docs/_data/atlantic-2023-config.yaml

```diff
@@ -6,7 +6,8 @@ city: Atlantic
 local_area: Atlantic
 area: Atlantic
 area_adj: Atlantic
-tz: CEST & EDT
+tz: UTC
+tz_header: CEST & EDT
```

Comment:
This is probably misleading and should be UTC? Or at least says UTC (CEST and EDT) or something

Reply:
We should have a blog post about the timezones and how they work out for various places

Reply:
Ah the tz_header is more: how we define the location in the header of the page. The conference isn't really "in" a timezone after all, but it fits CEST/EDT. Then the actual times of start/end of things is in UTC.

Reply:
yeah I know, I'm just worried about people getting confused.  If we're saying it's in UTC throughout (and all times on the site are in UTC) it might be better?

Reply:
Hmm I see the point. But if we say UTC I am concerned about missing audiences because UTC is kind of (even if incorrectly) associated with the UK. And "UTC (CEST and EDT)" feels more confusing.

Reply:
Agreed. `UTC (perfect for CEST, EDT and many others)` (not a serious suggestion)